awx/ui_next/src/screens/Inventory/InventoryGroupHosts/InventoryGroupHostList.jsx

```diff
@@ -16,6 +22,8 @@ import ErrorDetail from '../../../components/ErrorDetail';
 import PaginatedDataList from '../../../components/PaginatedDataList';
 import AssociateModal from '../../../components/AssociateModal';
 import DisassociateButton from '../../../components/DisassociateButton';
+import { Kebabified } from '../../../contexts/Kebabified';
```

Comment:
I tried using the hook and was not able to make that work. The problem that I ran into was the Kebabified is not true until you open the kebab icon.  At that point the drop down items are already rendered.

Reply:
You're right. We can't grab the values at the top of this component because it is not nested within the Kebabified Provider. I understand why you used this pattern now. 👍